.github/workflows/CICD.yml

```diff
@@ -0,0 +1,298 @@
+          - 8.0
+          - 8.1
+        sql-versions:
+          - mariadb
+          - postgresql
+          - sqlite
+    # Service containers to run with `container-job`
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+    env:
+      extensions: bcmath, curl, dom, gd, imagick, json, libxml, mbstring, pcntl, pdo, pdo_sqlite, pdo_mysql, pdo_pgsql, pgsql, sqlite3, zip
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set Up Imagick, FFmpeg & Exiftools
+        run: |
+          sudo apt-get update
+          sudo apt-get --fix-broken install
+          sudo apt-get -y install ffmpeg libimage-exiftool-perl
+
+      - name: setup Databases
+        run: |
+          sudo service mysql start
+          touch database/database.sqlite
+          mysql -uroot -proot -e 'create database homestead_test;'
+
+      - name: Setup PHP Action
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          extensions: ${{ env.extensions }}
+          tools: pecl, composer
+
+      - name: Install Composer dependencies
+        uses: ramsey/composer-install@v2
+        with:
+          composer-options: --no-dev
+
+      - name: Build Dist
+        run: |
+          make dist-clean
+
+      - name: copy Env
+        run: |
+          cp .github/workflows/.env.${{ matrix.sql-versions }} Lychee/.env
+
+      - name: setup MySQL Database with v3
+        run: |
+          mysql -uroot -proot homestead_test < .github/workflows/v3.sql
+
+      - name: Generate secure key & Migrate & go backward
+        run: |
+          cd Lychee
+          pwd
+          php artisan key:generate
+          php artisan migrate
+          php artisan migrate:rollback
+
+  createArtifact:
+    name: 3️⃣ Build Artifact
+    needs:
+      - phpstan
+      - dist
+      - tests
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'push' && github.event.pull_request.head.repo.full_name != github.repository && startsWith(github.ref, 'refs/heads/master'))
+    env:
+      extensions: bcmath, curl, dom, gd, imagick, json, libxml, mbstring, pcntl, pdo, pdo_sqlite, pdo_mysql, pdo_pgsql, pgsql, sqlite3, zip
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set Up Imagick, FFmpeg & Exiftools
+        run: |
+          sudo apt-get update
+          sudo apt-get --fix-broken install
+          sudo apt-get -y install ffmpeg libimage-exiftool-perl
```

Comment:
Why is this necessary for creating a ZIP archive? We don't run Lychee, we only want to pack it.